perun-core/src/main/java/cz/metacentrum/perun/core/implApi/modules/pwdmgr/PasswordManagerModule.java

```diff
@@ -70,4 +69,8 @@ default String handleSponsorship(PerunSession sess, SponsoredUserData userData)
 
 	String generateRandomPassword(PerunSession sess, String login);
 
+	default boolean loginExist(PerunSession sess, String login) {
+		return false;
```

Comment:
Maybe it would be better to throw an exception instead of false. Something like `OperationNotSupportedException`.
We have this method in OpenAPI so anybody can call it for various namespaces and we do not want to give them false information.

Reply:
Makes sense, changed for exception.